### PR TITLE
Adding a i18n warning to the user on cloud provider upload page to let t...

### PR DESCRIPTION
**Pull request by cam156**
...he user know the uploading is time sensitive.

screen shot 2015-03-05 at 8 59 08 am

### CODE REVIEW

**Comment by jcoyne** [comment]
@cam156 I'm wondering if "quickly" will confuse the users. They will wonder how they can control the rate they are transmitting at. How about "in a short timespan"

File: app/views/generic_files/_browse_everything.html.erb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <%= render partial: 'generic_files/upload/agreement' %>
+<div class="alert alert-success">
+  <%= t("sufia.upload.cloud_timeout_message") %>
```
Comment (mjgiarlo) [inline app/views/generic_files/_browse_everything.html.erb]:
```ruby
<%= t("sufia.upload.cloud_timeout_message", contact_href: link_to(t("sufia.upload.alert.contact_href_text"), sufia.contact_form_index_path)) %>
```

File: config/locales/sufia.en.yml [hunk 2]
```diff
@@ -53,6 +53,7 @@ en:
         contact_href_text: "contact form"
         partial_fail_html: "One or more files did not upload successfully. To continue using the files that uploaded %{metadata_href}. <br /> Use the %{contact_href} to report the error if it persists."
         partial_fail_metadata_href_text: "edit their metadata"
+      cloud_timeout_message: "Uploading from cloud providers is time sensitive.  Please do not upload large quantities of files quickly or the system may not be able to keep up.  If you do, you run the risk of getting an error message for your content instead of your desired content."
```
Comment (mjgiarlo) [inline config/locales/sufia.en.yml]: Recommend shortening this to:

```yaml
cloud_timeout_message: "Please note that if you upload a large number of files within a short period of time, the cloud provider may not be able to accommodate your request. If you experience any errors uploading from the cloud, let us know via the %{contact_href}."
```

File: spec/views/generic_file/_browse_everything.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe 'generic_files/_browse_everything.html.erb', :type => :view do
+  it 'shows user timing warning' do
+    render
+    page = Capybara::Node::Simple.new(rendered)
+    expect(page).to have_selector('div.alert-success', text: /Uploading from cloud providers is time sensitive/i , count: 1)
```
Comment (mjgiarlo) [inline spec/views/generic_file/_browse_everything.html.erb_spec.rb]: Change text to "Please note that if you upload a large number of files" per above?

**Pull request merged**